.gitignore

```diff
@@ -4,7 +4,7 @@
 #Ignore folders
 node_modules
 scrap
-public/dist
+client/dist
```

Comment:
I'm just curious why you added this folder to ignore?

Reply:
I don't want to include my dist folder because it can be built from everything else so therefore just takes up space it doesn't need to. My understanding is it's best practice to not include the bundle in your commits.